src/printer.js

```diff
@@ -3880,7 +3880,29 @@ function makeString(rawContent, enclosingQuote) {
     return match;
```

Comment:
Huh, how about that! Fixed in https://github.com/prettier/prettier/pull/1575/commits/8934399a5a0587c4b12b4e5c4c5e39bf6fc3e21b

I don't yet completely understand how this works without using lookbehind, but it passes the tests.